.eslintrc.js

```diff
@@ -38,10 +38,48 @@ module.exports = {
     },
   },
   rules: {
+    'comma-style': 2,
```

Comment:
prettier autoformats this file when you run the devserver. maybe add a `// prettier-ignore` comment or run it through the devserver once to get the changes

Reply:
What needs formatting?

Reply:
the eslintrc.js file itself